host/3.0/buster/amd64/python/python36/python36-buildenv.Dockerfile

```diff
@@ -10,7 +10,10 @@ RUN apt-get update && \
        libncurses5-dev libncursesw5-dev xz-utils tk-dev && \
     tar xzf Python-3.6.15.tgz && \
```

Comment:
Please remove the tar file once its work is done.

I would recommend un-tarring this to `/usr/src/python` or such a directory as we don't want to create folders with the version name as well.

Why don't we copy the whole https://github.com/docker-library/python/blob/222c5a1347165e8f6a691353ab3646dce73497ce/3.6/buster/slim/Dockerfile#L28-L144 section to make it exact copy of what docker/3.6 did?